submissions/zalevskyi/dom/js/app.js

```diff
@@ -0,0 +1,124 @@
+        img_src: './img/Pingala.png',
+        img_alt: 'Pingala portrait'
+    },
+    'reyna' : {
+        type: 'governor',
+        title: 'The Financier',
+        description: 'Shrewd and fiscally knowledgeable, Reyna is able to strengthen the economic power of a city, using finances to provide more than is normally possible.',
+        context: 'A financier is an official tasked with resolving budgetary and financial matters, requiring an astute understanding of economics and the psychology of markets. Playing with the money of the state is not a matter for the faint-hearted.',
+        img_src: './img/Reyna.png',
+        img_alt: 'Reyna portrait'
+    },
+    'victor' : {
+        type: 'governor',
+        title: 'The Castellan',
+        description: 'Military focused, once established in a city, Victor is instrumental in strengthening, preserving, and defending it from foreign threats.',
+        context: 'A Castellan is a military officer responsible for directing the day-to-day affairs of a castle or fortress and the territory immediately around it. As castles were massive investments of treasure and important points of military power, the castellan had to be both competent and politically reliable. In times of war, a competent castellan could determine the fate of campaigns, and with that, the historical outcome.',
+        img_src: './img/Victor.png',
+        img_alt: 'Victor portrait'
+    }
+}
+Object.freeze(WIKI)
+const MAIN_INNER_HTML = `<input type='button' id='menu_button' class='menu_button' value='Menu'>
+                         <div id='civilopedia_path' class='civilopedia_path'>Civilization VI | Governors</div>
+                         <div id='article' class='article'><h1 id='title' class='text_heading_1'></h1>
+                            <img id='portrait' alt='' src='' class='portrait'>
+                            <h2 class='text_heading_2 description_heading'>Description</h2>
+                            <p id='description' class='text description'></p>
+                            <h2 class='text_heading_2 context_heading'>Historical Context</h2>
+                            <p id='context' class='text context'></p>
+                         </div>`
+const main = document.getElementById('main')
+const nav = document.getElementById('nav')
+main.innerHTML=MAIN_INNER_HTML
+const title = document.getElementById('title')
+const description = document.getElementById('description')
+const portrait = document.getElementById('portrait')
+const context = document.getElementById('context')
+const governorHeadings = document.querySelectorAll('.text_heading_2')
+hideGovernorDetails()
```

Comment:
you can divide your constants functions declarations and it's calls by adding empty lines. It can increase readability of your code for others developers.